logbook-spring/src/main/java/org/zalando/logbook/spring/RemoteResponse.java

```diff
@@ -62,6 +62,7 @@ public State without() {
         @Override
         public State buffer(final ClientHttpResponse response) throws IOException {
             InputStream responseBodyStream = response.getBody();
+            responseBodyStream.mark(Integer.MAX_VALUE);
```

Comment:
Why is that needed?

Reply:
Doesn't a stream need to be marked before it can be reset?

Reply:
You're probably right. Interesting that this never caused any issue before, since we had `reset` in there for a while.